package vdi.lane.reconciliation

internal class CriticalReconciliationError : Exception()
